### PR TITLE
PLpeople layer info

### PR DESCRIPTION
**Pull request by nstjean**
Continues https://github.com/publiclab/leaflet-environmental-layers/pull/361

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by nstjean** [comment]
FireShot Capture 340 - Leaflet Environmental Layers -

I think the way the disclaimers are shown could be improved... I'm not sure showing up in the middle of the screen is ideal, nor the way it doesn't go away without clicking on the "close" link.

**Comment by crisner** [comment]
Agreed! They need to be improved. I am thinking it is best to remove the current way it is implemented and use bootstrap modals instead since the layer menu already used bootstrap.

**Comment by nstjean** [comment]
Ok got this passing tests now. @crisner can you review?

**Comment by nstjean** [comment]
> Agreed! They need to be improved. I am thinking it is best to remove the current way it is implemented and use bootstrap modals instead since the layer menu already used bootstrap.

Yes I think that would look much better!

File: src/info.json [hunk 1]
```diff
@@ -319,9 +319,9 @@
     "url": "http://publiclab.org",
     "data": {
       "type": "",
-      "disclaimer": "disclaimer"
+      "disclaimer": "Learn more about our method of <a href='https://publiclab.org/wiki/location-privacy/'>location privacy</a> and how our <a href='https://publiclab.org/notes/sagarpreet/04-02-2019/introducing-leaflet-blurred-location-display'>blurred location library</a> works."
```
Comment (crisner) [inline src/info.json]: Looks good to me.

**Pull request merged**